src/viam/examples/modules/mlmodelservice_tflite.cpp

```diff
@@ -0,0 +1,759 @@
+#include <pthread.h>
+#include <signal.h>
+
+#include <fstream>
+#include <iostream>
+#include <mutex>
+#include <sstream>
+#include <stdexcept>
+
+#include <grpcpp/channel.h>
+#include <grpcpp/create_channel.h>
+#include <grpcpp/security/credentials.h>
+#include <tensorflow/lite/c/c_api.h>
+
+#include <viam/sdk/components/component.hpp>
+#include <viam/sdk/config/resource.hpp>
+#include <viam/sdk/module/service.hpp>
+#include <viam/sdk/registry/registry.hpp>
+#include <viam/sdk/rpc/server.hpp>
+#include <viam/sdk/services/mlmodel/mlmodel.hpp>
+#include <viam/sdk/services/mlmodel/server.hpp>
+
+namespace {
+
+namespace vsdk = ::viam::sdk;
+constexpr char service_name[] = "mlmodelservice_tflite";
+
+class MLModelServiceTFLite : public vsdk::MLModelService {
```

Comment:
This should have a class level comment that gives an overview and also documents the configuration parameters.